public/app/core/components/AppChrome/NavToolbar.tsx

```diff
@@ -37,19 +38,31 @@ export function NavToolbar({
 
   return (
     <div className={styles.pageToolbar}>
-      <div className={styles.menuButton}>
-        <IconButton name="bars" tooltip="Toggle menu" tooltipPlacement="bottom" size="xl" onClick={onToggleMegaMenu} />
-      </div>
-      <Breadcrumbs breadcrumbs={breadcrumbs} className={styles.breadcrumbs} />
+      {!config.isPublicDashboardView && (
+        <>
+          <div className={styles.menuButton}>
+            <IconButton
+              name="bars"
+              tooltip="Toggle menu"
+              tooltipPlacement="bottom"
+              size="xl"
+              onClick={onToggleMegaMenu}
+            />
+          </div>
+          <Breadcrumbs breadcrumbs={breadcrumbs} className={styles.breadcrumbs} />
```

Comment:
Breadcrumbs show dashboard title which are an important context to show what your looking at. Wonder how we can do this for public dashboard.